message ReturnValue {
	required string value = 1; 
}